.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
+count = True
+show-source = True
+statistics = True
+indent-size = 2
+max-line-length = 128
```

Comment:
I would prefer 100 or so, for more manageable side-by-side diffs, and for working with two files at a time.

Reply:
100 SGTM (I had just copied 128 from the workflow config). Done in https://github.com/unicode-org/unicodetools/pull/158/commits/246732e8022269e63cde306c79d7fa957d46f459